requests/sessions.py

```diff
@@ -715,6 +716,10 @@ def merge_environment_settings(self, url, proxies, stream, verify, cert):
             # Look for requests environment configuration and be compatible
             # with cURL.
             if verify is True or verify is None:
+                if os.environ.get('REQUESTS_CA_BUNDLE'):
+                    warnings.warn('Setting verify value to environment value of REQUESTS_CA_BUNDLE')
+                if os.environ.get('CURL_CA_BUNDLE'):
```

Comment:
Assuming we wanted to get more harassment from our users over warning messages... this should be an `elif`. If both are set we're emitting both warnings for the same request which will confuse users because we're going to prefer `REQUESTS_CA_BUNDLE` over `CURL_CA_BUNDLE` and so should only warn once with the real informatoin

Reply:
Oh, yes. Sorry my bad.